app/scripts/views/support/create_secure_password.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const View = require('views/base').extend({
```

Comment:
It's valid syntax, but I find the `require('views/base')` a bit strange since we don't do that anywhere else.